app.go

```diff
@@ -62,6 +62,29 @@ func newApp(broker *Broker, mode, route, addr, keyID, keySecret string) *App {
 	}
 }
 
+func (app *App) disconnect(clientID string) error {
+	req, err := http.NewRequest("POST", app.addr+"/disconnect", nil)
+	if err != nil {
+		return fmt.Errorf("failed creating request: %v", err)
+	}
+
+	req.SetBasicAuth(app.keyID, app.keySecret)
+	req.Header.Set("Wave-Client-ID", clientID)
+
+	resp, err := app.client.Do(req)
+	if err != nil {
+		return fmt.Errorf("request failed: %v", err)
+	}
+	defer resp.Body.Close()
+	if resp.StatusCode != 200 {
```

Comment:
I would say it's not a big deal since we are in charge of sending the response from python which is only capable of sending 200, 401 or 400 meaning even if we allow full 2xx range, it would never execute beyond 200.